test/rendering/TestApp.jsx

```diff
@@ -65,7 +65,8 @@ class TestApp extends React.Component {
       users: [
           {id: "http://fel.cvut.cz/people/max-chopart", label: "Max Chopart"},
           {id: "http://fel.cvut.cz/people/miroslav-blasko", label: "Miroslav Blasko"}],
-      currentUser: "http://fel.cvut.cz/people/max-chopart"
+      currentUser: "http://fel.cvut.cz/people/max-chopart",
+      enableComments: false
```

Comment:
I suggest :

questionComments: enable | disable | <onHover> 
questionHelp: <enable> | disable | onHover 

while <> is default value.

Reply:
That if developer does not set up the value it should behave like that.